assets/app/view/game_page.rb

```diff
@@ -53,20 +53,7 @@ def load_game
       return if game_id == @game&.id &&
         ((!cursor && @game.actions.size == @num_actions) || (cursor == @game.actions.size))
 
-      # Some Hotseat games don't have player ids, use names instead.
-      players = @game_data['players'].map { |p| [p['id'] || p['name'], p['name']] }.to_h
-      # Back compatibility, make hotseat games continue to work after the change to play names
-      if actions&.first&.dig(:entity_type) == 'player' && actions.first[:entity].is_a?(String)
-        players = @game_data['players'].map { |p| [p['name'], p['name']] }.to_h
```

Comment:
Is it important to keep this part for backward compatibility?

Reply:
na